rhel7/fixes/ansible/container_all_services_disabled.yml

```diff
@@ -0,0 +1,9 @@
+# platform = multi_platform_rhel,multi_platform_fedora
+# reboot = true
+# strategy = restrict
+# complexity = low
+# disruption = low
+- name: "Disable all enabled services except for systemd, getty, and autovt services"
+  shell: systemctl list-unit-files --type=service | grep enabled | grep -oP '^(?!systemd-)(?!getty)(?!autovt)\S+\.service' | xargs systemctl disable
```

Comment:
It is worth doing some research whether we can use other means than the `shell` module.